airflow/jobs/local_task_job.py

```diff
@@ -264,3 +265,15 @@ def _run_mini_scheduler_on_child_tasks(self, session=None) -> None:
                 exc_info=True,
             )
             session.rollback()
+
+    @provide_session
+    def _update_dagrun_state_for_paused_dag(self, session=None):
+        """
+        Checks for paused dags with DagRuns in the running state and
+        update the DagRun state if possible
+        """
+        dag_run = self.task_instance.get_dagrun()
+        if dag_run:
+            dag = dag_run.dag = self.task_instance.task.dag
+            if dag.get_is_paused():
+                dag_run.update_state(session=session, execute_callbacks=False)
```

Comment:
If we say `execute_callbacks=False` here, then this means that the return value will contain a list of callbacks -- so I think we want True here.